editor/constants.js

```diff
@@ -0,0 +1,10 @@
+import scssVariables from '!!sass-variables-loader!./assets/stylesheets/_variables.scss';
+
+export { scssVariables };
```

Comment:
We use only `BREAK_MEDIUM` in the code. Do we really need to expose all other variables? This list is quite huge, see: https://github.com/WordPress/gutenberg/blob/master/editor/assets/stylesheets/_variables.scss.

Reply:
Regarding the other BREAK... constants we are creating there is a pull request where we add responsive-redux and I plan to use this variable as our breakpoints.
Regarding the export of scssVariables that's something we should discuss I think exposing it may be useful in the future there are situations where we may need to access a color or something similar. But we may decide that it is better to create constants for this cases like we are doing here to have a better control.

Reply:
> Regarding the export of scssVariables that's something we should discuss I think exposing it may be useful in the future there are situations where we may need to access a color or something similar.

It might, it might not :) My point of view here is as follows, it can be always added later when it's really needed. I try to follow [YAGNI rule](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) in such cases.

Reply:
Yes that's true given that right now we don't have an external usage for scssVariables I followed your suggestion and removed the export :)